src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -4,25 +4,44 @@
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix6.hardware.TalonFX;
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import com.revrobotics.CANSparkMax;
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.Constants.IntakeConstants;
 import frc.robot.stateManagement.PlacementMode;
 
 public class IntakeSubsystem extends SubsystemBase {
-  private CANSparkMax intakeMotor;
+  private TalonFX intakeMotor;
   private CANSparkMax chooserMotor;
+  private GenericEntry intakeOutput;
+  private GenericEntry chooserOutput;
+  private ShuffleboardTab intakeTab = Shuffleboard.getTab("Intake");
 
   public IntakeSubsystem() {
-    intakeMotor = new CANSparkMax(Constants.IntakeConstants.INTAKE_MOTOR_ID, MotorType.kBrushless);
-    chooserMotor = new CANSparkMax(Constants.IntakeConstants.INTAKE_CHOOSER_ID, MotorType.kBrushed);
-    intakeMotor.restoreFactoryDefaults();
+    intakeMotor = new TalonFX(Constants.IntakeConstants.INTAKE_MOTOR_ID, "Default Name");
```

Comment:
`"Default Name"`?

Reply:
Disregard.

Reply:
Per Casey this is the name Phoenix Tuner uses for the CAN bus, and we probably can't change it.